Source/CLI/rawcooked.1

```diff
@@ -207,6 +202,13 @@ Default value is \fI${Input}.framemd5\fR.
 Do not compute of framemd5 of input frames. (see above)."
 .br
 Is default.
+.TP
+.B --accept-gaps
+Use if there are missing files within the sequence numbering. RAWcooked creates a concatenated list of all files ensuring the sequence can be encoded.
+.TP
+.B --no-accept-gaps
+Do not accept-gaps within the sequence numbering. FFmpeg will fail any encoding attempts where gaps are present.
+.TP
```

Comment:
@JeromeMartinez Should one of `--accept-gaps` and `--no-accept-gaps` not be default?

Reply:
Ah thanks @retokromer I missed that!

Reply:
@retokromer I added the default value (`--no-accept-gaps`).
@digitensions I combined in one commit, resolved the merge conflict, added the default value, and forced-pushed the commit to your branch and this PR. Let me know if something is missing or has an issue.